src/main/webapp/js/adminHome.js

```diff
@@ -30,6 +30,41 @@ function createRowForResultTable(shortName, name, email, institution, isSuccess,
     return result;
 }
 
+/**
+ * Fully encodes a string of instructor details in the form
+ * [name] | [email] | [institution]
+ *
+ * e.g. 'Test Instructor | test.instructor@gmail.com | Random Institution'
+ *
+ * @param {String} detailsString is the string of details as above
+ * @returns {String} the encoded form of detailsString
+ */
+function encodeInstructorDetails(detailsString) {
+    return encodeURIComponent(detailsString);
+}
+
+/**
+ * Fully encodes an arbitrary query string, but skips the
+ * = and & characters used to define and separate parameters.
+ *
+ * e.g.
+ * 'instructorname=Test Instructor&instructoremail=test.instructor@gmail.com'
+ * becomes
+ * 'instructorname=Test%20Instructor&instructoremail=test.instructor%40gmail.com'
+ *
+ * @param queryString
+ * @returns the encoded query string
+ */
+function encodeQueryString(queryString) {
+    return queryString.split('&')
```

Comment:
What if the user input contains `&`? The data should be encoded before constructing the query string. Or you could use [`jQuery.param`](http://api.jquery.com/jQuery.param/) to do both together